app-android/src/androidTest/java/com/ustadmobile/port/android/DownloadJobConnectivityChangeTriggerEventsTest.kt

```diff
@@ -8,10 +8,7 @@ import com.ustadmobile.door.DoorObserver
 import com.ustadmobile.door.entities.NodeIdAndAuth
 import com.ustadmobile.door.ext.DoorTag
 import com.ustadmobile.door.util.randomUuid
-import com.ustadmobile.lib.db.entities.ConnectivityStatus
-import com.ustadmobile.lib.db.entities.ContentEntry
-import com.ustadmobile.lib.db.entities.DownloadJob
-import com.ustadmobile.lib.db.entities.DownloadJobItem
+import com.ustadmobile.lib.db.entities.*
```

Comment:
I'm not sure what this test is really testing. As far as I can see, it is really only testing livedata. Not sure where this test came from, but it seems pointless. Also it's waiting quite a long time, looks like 10secs+ because of thread.sleep